launcher/ui/dialogs/OfflineLoginDialog.cpp

```diff
@@ -0,0 +1,113 @@
+/* Copyright 2013-2021 MultiMC Contributors
```

Comment:
Change this to "Copyright 2021 PolyMC Contributors"

Not sure if the license thing is needed for GPL but just remove it for now

Reply:
The entire header could be removed actually, does that work for you?

Reply:
Sure!